docs/adr/0008-build-off-lbh-frontend-rather-than-govuk-react.md

```diff
@@ -0,0 +1,21 @@
+# 7. Build off lbh-frontend rather than govuk-react
+
+Date: 2019-10-30
+
+## Status
+
+Accepted
+
+## Context
+
+We need to be consitent with the Hackney styles within lbh-frontend. We could derive styles from lbh-frontend, while reimplementating the HTML and JS. We could also use govuk-react, a repository pre-built that derives from govuk-frontend, and then apply the modifications from lbh-frontend to it. The repository for govuk-react may not be maintained given that it has not received updates recently despite having a large base of issues raised against it. It is also built using
+styled-components for styling, which creates technical difficulty given that lbh-frontend is built using SASS.
```

Comment:
I would really like to validate this assertion. Can we try and see if we can use `govuk-react` and `lbh-frontend` together? It would save us a lot of effort to be able to wrap those components with some extra CSS.

Reply:
I think it's worth spending some time on investigating this, so we can be really confident this is the best approach.